index.html

```diff
@@ -1,23 +1,491 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title>Document</title>
-        <link rel="stylesheet" href="./styles.css" />
-    </head>
-    <body>
-        <div class="header">
-            <div class="header__logo">{Finsweet</div>
-            <div class="header__links">
-                <a href="#" class="header__link--home"> Home</a>
-                <a href="#" class="header__link--blog">Blog</a>
-                <a href="#" class="header__link--about">About us</a>
-                <a href="#" class="header__link--contact">Contact us</a>
-            </div>
-            <button class="header__subcribe">Subcribe</button>
-            <button class="jsakldjksadjklas">asdjlksajkd</button>
-        </div>
-    </body>
-</html>
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" heroinfo="IE=edge" />
+    <meta name="viewport" heroinfo="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./styles.css" />
+    <style>
+
+    </style>
+</head>
+
+<body>
+    <div class="homepage">
+        <!--  -->
+        <div class="homepage__header">
+            <div class="homepage__logo">
+                {Finsweet
+            </div>
+            <div class="homepage__links">
+                <a href="#" class="homepage__link--home">Home</a>
+                <a href="#" class="homepage__link--blogs">Blogs</a>
+                <a href="#" class="homepage__link--about">About us</a>
+                <a href="#" class="homepage__link--contact">Contact us</a>
+            </div>
+            <button class="homepage__btnSubmit">Subcribe</button>
+        </div>
+
+        <!--  -->
+        <div class="homepage__homehero">
```

Comment:
Đặt tên class tách riêng các từ ra, sửa các lỗi tương tự phía bên dưới luôn em nhé
```suggestion
<div class="homepage__home-hero">
```